cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js

```diff
@@ -80,7 +80,11 @@ export default {
   actionsIsAbsent() {
     cy.expect(actions.absent());
   },
-
+  verifyPopulatedPreviewPage() {
```

Comment:
Please, use array of assertions
Example:
 cy.expect([
      deleteUploadedFileModal.exists(),
      deleteUploadedFileModal.find(HTML(including(modalContent))).exists(),
      deleteUploadedFileModal.find(modalNoButton, { disabled: true }).exists(),
      deleteUploadedFileModal.find(modalYesButton, { disabled: false }).exists(),
    ]);